package.json

```diff
@@ -27,6 +27,7 @@
   ],
   "scripts": {
     "build": "tsup && yarn build:types",
+    "build:clean": "rimraf dist && yarn build",
```

Comment:
I don't see `rimraf` in the dependency list

Reply:
There is a `--clean` flag for `tsup` that we could consider using instead of rimraf

Reply:
`--clean` seems to do what we want. I've made that change here: [d04b7e7](https://github.com/MetaMask/metamask-module-template/pull/238/commits/d04b7e7b90e988482a08fa8951c6e30b957a4589)